tools/src/test/python/dlpx/virtualization/_internal/test_plugin_validator.py

```diff
@@ -118,6 +118,7 @@ def test_plugin_additional_properties(plugin_config_file,
     @staticmethod
     @pytest.mark.parametrize('host_types', [['xxx']])
     @pytest.mark.parametrize('src_dir', [None])
+    @pytest.mark.parametrize('plugin_type', ['INDIRECT'])
```

Comment:
not really your change but is there a reason these are parametrized separately? if there was more than one I'd get it (it does a cross of every possible parameterization,) but since there's only one value for any of these keys could we not do it in one?